client/wfnews-war/src/main/angular/src/app/components/sticky-widget/contact-widget-dialog/contact-widget-dialog.component.html

```diff
@@ -16,14 +16,13 @@ <h1 mat-dialog-title>Contact Information <mat-icon [color]="closeColor"
                     <div class="form">
                         <div class="row">
                             <div class="field">
-                                <mat-label class="label required">Name</mat-label>
+                                <mat-label class="label required">Name<span style="color: red">*</span></mat-label>
                                 <input matInput type="text" formControlName="name" aria-label="Name" />
-                                <mat-error [style.visibility]="error('name', 'required') ? 'visible':'hidden'">Name is required</mat-error>
                             </div>
                             <div class="field">
-                                <mat-label class="label  required">Email</mat-label>
+                                <mat-label class="label required">Email<span style="color: red">*</span></mat-label>
                                 <input  matInput type="text" formControlName="email" aria-label="Email" />
-                                <mat-error>{{ mailErrorMessages }}</mat-error>
+                                <mat-error style="font-size: 15px; font-size: 15px; padding-top: 7px; ">{{ mailErrorMessages }}</mat-error>
```

Comment:
there are two "font-size" here, please remove one of thme

Reply:
Whoops, Removed!